src/main/kotlin/org/jetbrains/research/testspark/helpers/LLMHelper.kt

```diff
@@ -66,6 +67,9 @@ object LLMHelper {
             if (platformSelector.selectedItem!!.toString() == settingsState.grazieName) {
                 modules = getGrazieModels()
             }
+            if (platformSelector.selectedItem!!.toString() == settingsState.huggingFaceName) {
+                modules = getHuggingFaceModels()
```

Comment:
We likely had a typo in the variable name; it should have been `models`. I created an [issue](https://github.com/JetBrains-Research/TestSpark/issues/268) for this. You may either leave it as is or fix it if you want to.